cli/src/api-key-success.html

```diff
@@ -0,0 +1,39 @@
+<html>
+  <head>
+    <title>API Key generated successfully</title>
+    <style>
+      body {
+        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+          'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+
+        text-align: center;
+        margin-top: 48px;
+      }
+    </style>
+  </head>
+  <body>
+    <div style="width: 200px; margin: 0 auto">
+      <svg viewBox="0 0 143 41" focusable="false" class="chakra-icon chakra-icon css-1aptyfp">
```

Comment:
```
<svg aria-hidden="true" focusable="false">
  <!-- svg content -->
</svg>
```

would make more sense, focusable="false" is just IE hack so we also need to hide the icon with aria-hidden as it's pure decorative.

The other option:
```
<svg class="critter" viewBox="0 0 100 100" aria-labelledby="uniqueTitleID" aria-describedby="uniqueDescID" role="img">
  <title id="uniqueTitleID">Baxter</title>
  <desc id="uniqueDescID">A happy little bear</desc>
</svg>
```

Reply:
Cool. I've added `aria-hidden` here https://github.com/xataio/client-ts/pull/367/commits/72a8371446c35baf4e19367f58831eba7a2dbd9a based on your first suggestion.